OpenRA.Mods.Common/Traits/Render/WithAttackAnimation.cs

```diff
@@ -47,13 +54,25 @@ public WithAttackAnimation(ActorInitializer init, WithAttackAnimationInfo info)
 			wsb = init.Self.Trait<WithSpriteBody>();
 		}
 
-		public void Attacking(Actor self, Target target, Armament a, Barrel barrel)
+		void INotifyAttack.Attacking(Actor self, Target target, Armament a, Barrel barrel)
+		{
+			if (info.DelayRelativeTo == AttackDelayType.Attack && !string.IsNullOrEmpty(info.AttackSequence))
+				self.World.AddFrameEndTask(w => w.Add(new DelayedAction(info.Delay, () =>
+					{
```

Comment:
Cases like this are why we have a(n often ignored) rule to avoid hanging blocks.  When you have an inner block with braces all outer blocks should also have braces, so please add them here.